src/main/java/com/altenheim/kalender/controller/logicController/IOController.java

```diff
@@ -41,77 +38,45 @@ public IOController(IEntryFactory administrateEntries, List<ContactModel> allCon
         this.settings = settings;
         this.mailTemplates = mailTemplates;
     }
-    
 
-    public void writeCalendarFiles() throws ValidationException, IOException
+
+    public void saveDecryptedPasswordHash(String hashedPassword)
     {
-        var allCalendars = allEntries.getAllCalendars();
-        for (var calendarSet : allCalendars) 
-        {
-            var icsCalendar = new Calendar();
-            icsCalendar.getProperties().add(new ProdId("-//Smart Planner//iCal4j 1.0//DE"));
-            icsCalendar.getProperties().add(Version.VERSION_2_0);
-            icsCalendar.getProperties().add(CalScale.GREGORIAN);
-            //icsCalendar.getProperties().add(new Name(calendarSet.getKey()));
-            /*for (var entry : calendarSet.clear();)        
-                icsCalendar.getComponents().add(createIcalEntryFromCalFXEntry(entry));        
-            var fout = new FileOutputStream("icsFiles/" + calendarSet.getKey() + ".ics");
-            var outputter = new CalendarOutputter();
-            outputter.output(icsCalendar, fout);
-            fout.close();     */     
-        }        
+        this.hashedPassword = hashedPassword;
     }
+    public String getDecryptedPasswordHash() { return hashedPassword; }
 
+    public void writeCalendarFiles() throws ValidationException, IOException
+    {
 
-    public void loadCalendarsFromFile() throws IOException, ParserException
+    }
+
+    public void createUserPath()
     {
-        var folder = new File("icsFiles");
-        var files = folder.listFiles();
-        var calendarSource = new CalendarSource("Saved Calendars");
-        var calendars = new ArrayList<com.calendarfx.model.Calendar>();
+        var parentFolder = new File("userFiles");
```

Comment:
Hier werden Ordner nach der Struktur im Verzeichnis erzeugt, in der das Programm gestartet wird. Wird dann später noch angepasst sein an das Standard bzw vom User definierte Verzeichnis (währned des initialen Setups und/oder in den Settings).